pyctcdecode/tests/test_decoder.py

```diff
@@ -512,6 +512,51 @@ def test_beam_results(self):
         self.assertEqual(beams[0].text, "bugs bunny")
         self.assertEqual(beams[1].text, "bunny bunny")
 
+    def test_partial_decode(self):
```

Comment:
Could we add a test or two for the hotword scorer stuff? Like maybe verify that we get the same results for a logits slice when providing the same hotword scorer, but different ones when providing different ones over the course of partial decoding?